packages/api/tests/endpoints/test-granules.js

```diff
@@ -1510,7 +1510,7 @@ test.serial('PUT with action move returns failure if more than one granule file
   filesExistingStub.restore();
 });
 
-test.serial('POST creates new granule in dynamoDB and postgres', async (t) => {
+test.serial('create (POST) creates new granule in dynamoDB and postgres', async (t) => {
   const newGranule = fakeGranuleFactoryV2({
     collectionId: t.context.collectionId,
     execution: undefined,
```

Comment:
Do we want to test the case when `execution` is defined? For both `create` and `update`.

Reply:
fcf8a2363f